src/cohortmanager.js

```diff
@@ -0,0 +1,156 @@
+const cohortNotFoundError = 'Cohort not found.'
+const cohortFullError = 'This cohort is full.'
+const cohortCannotExistWithoutNameError =
+  'Please name the cohort before initialising.'
+const cohortAlreadyExistsError = 'This cohort already exists.'
+const studentNotFoundError = 'Student not found.'
+const studentInAnotherCohortError = 'This student is already in another cohort.'
+
+class CohortManager {
+  constructor() {
+    this.studentList = []
+    this.schoolCohorts = []
+    this.cohortCapacity = 24
+  }
+
+  createStudent(student) {
+    this.studentList.push(student)
+  }
+
+  createNStudents(num, student) {
+    for (let i = 0; i < num; i++) {
+      this.studentList.push(student)
+    }
+  }
+
+  createCohort(cohortName) {
+    const exists = this.checkIfCohortExists(cohortName)
+    if (cohortName === undefined) {
+      return cohortCannotExistWithoutNameError
+    } else if (exists) {
+      return cohortAlreadyExistsError
+    }
+    const cohort = {
+      name: cohortName,
+      students: [],
+      cohortCapacity: this.cohortCapacity
+    }
+    this.schoolCohorts.push(cohort)
+    return cohort
+  }
+
+  checkIfCohortExists(cohortName) {
+    if (this.searchCohort(cohortName) !== cohortNotFoundError) {
+      return true
+    }
+    return false
+  }
+
+  removeCohort(cohortName) {
+    for (let i = 0; i < this.schoolCohorts.length; i++) {
+      const cohort = this.schoolCohorts[i]
+      if (cohort.name === cohortName) {
+        this.schoolCohorts.splice(i, 1)
+        return
+      }
+    }
+    return cohortNotFoundError
+  }
+
+  checkIfStudentAlreadyInCohort(studentFirstName, studentLastName) {
+    for (let i = 0; i < this.schoolCohorts.length; i++) {
+      for (const element of this.schoolCohorts[i].students) {
+        if (
+          element.studentFirstName === studentFirstName &&
+          element.studentLastName === studentLastName
+        ) {
+          return true
+        }
+      }
+    }
+    return false
+  }
+
+  addStudentToCohort(studentFirstName, studentLastName, cohortName) {
+    const cohort = this.searchCohort(cohortName)
+    const studentAlreadyInCohort = this.checkIfStudentAlreadyInCohort(
+      studentFirstName,
+      studentLastName
+    )
+    if (studentAlreadyInCohort) {
+      return studentInAnotherCohortError
+    }
```

Comment:
theres quite a few cases in your code where i believe you could make your if statements one line, which i personally find easier to read  eg here you could make it` if (studentAlreadyInCohort) return studentInAnotherCohortError`

but other than that i like the way you did this function